store/postgres/action.go

```diff
@@ -42,6 +43,14 @@ func (s Store) selectAction(ctx context.Context, id string, txn *sqlx.Tx) (model
 	var fetchedAction Action
 
 	err := s.DB.WithTimeout(ctx, func(ctx context.Context) error {
+		nr := newrelic.DatastoreSegment{
+			Product:    newrelic.DatastorePostgres,
+			Collection: fmt.Sprintf("actions"),
```

Comment:
Do we need `Sprintf` here? and should it be `Action` with uppercase `A` (similary like `Policy` above)?